src/Aspire.Hosting.MongoDB/MongoDBBuilderExtensions.cs

```diff
@@ -247,4 +259,63 @@ private static void ConfigureMongoExpressContainer(EnvironmentCallbackContext co
         context.EnvironmentVariables.Add("ME_CONFIG_MONGODB_URL", sb.ToString());
         context.EnvironmentVariables.Add("ME_CONFIG_BASICAUTH", "false");
     }
+
+    /// <summary>
+    /// Same as <see cref="ResourceBuilderExtensions.WaitFor{T}(IResourceBuilder{T}, IResourceBuilder{IResource})"/> but with a few options we need.
+    /// </summary>
+    private static IResourceBuilder<T> WaitFor<T>(this IResourceBuilder<T> builder, IResourceBuilder<IResource> dependency, bool includeHealthChecks = true) where T : IResource
```

Comment:
I'd rather make it possible to disable the health check in the WaitFor call than to copy all of this source code.

Reply:
Yeah, I agree - I can make that change as part of this